components/snackbar.css

```diff
@@ -0,0 +1,43 @@
+.snackbar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 2rem 0rem;
```

Comment:
you need not mention unit for 0 margin or padding(you can use "margin: 2rem 0;")

Reply:
Ok.